### PR TITLE
Add Categories.Regex

### PR DESCRIPTION
**Pull request by stephentoub**
@DrewScoggins, per our conversation yesterday, is this the right way to do this?

### CODE REVIEW

File: src/benchmarks/micro/libraries/System.Text.RegularExpressions/Perf.Regex.Industry.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public static int Count(Regex r, string input)
     }
 
     /// <summary>Performance tests adapted from https://github.com/mariomka/regex-benchmark</summary>
-    [BenchmarkCategory(Categories.Libraries, Categories.NoWASM)]
+    [BenchmarkCategory(Categories.Libraries, Categories.Regex, Categories.NoWASM)]
```
Comment (DrewScoggins) [inline src/benchmarks/micro/libraries/System.Text.RegularExpressions/Perf.Regex.Industry.cs:43]: You will need the Regex category for all of the test classes, unless you don't want them included

**Comment by DrewScoggins** [comment]
Only the one comment, other than that LGTM.

File: src/benchmarks/micro/libraries/System.Text.RegularExpressions/Perf.Regex.Industry.cs (hunk 1 above)
Comment (stephentoub) [reply src/benchmarks/micro/libraries/System.Text.RegularExpressions/Perf.Regex.Industry.cs:43]: Oops, thanks.

**Comment by stephentoub** [comment]
FYI, @danmoseley, @joperezr.

**Comment by stephentoub** [comment]
@DrewScoggins, fixed, thanks.

**Comment by danmoseley** [comment]
What cmd line do you use to run just these and how long does it take?

**Comment by stephentoub** [comment]
> What cmd line do you use to run just these and how long does it take?

I'm not sure.  That wasn't actually my goal here.  Rather, I asked Drew how we could get a daily/weekly/etc. report generated that shows the history of all regex tests in an easily browsable manner, and this was his recommendation.

**Review by danmoseley — APPROVED** [review]

**Comment by stephentoub** [comment]
I don't have permission to merge, if someone who does can do so.

**Comment by DrewScoggins** [comment]
Done.

Also @danmoseley to answer your question you just need to use the `--anyCategories` command line argument and specify Regex as the category.

**Comment by stephentoub** [comment]
Thanks, @DrewScoggins.

**Pull request merged**